backend/src/graphql/models/GdtEntryList.ts

```diff
@@ -0,0 +1,56 @@
+import { GdtEntry } from './GdtEntry'
+import { ObjectType, Field } from 'type-graphql'
+
+@ObjectType()
+export class GdtSumPerEmail {
+  constructor(email: string, summe: number) {
+    this.email = email
+    this.summe = summe
+  }
+
+  @Field(() => String)
+  email: string
+
+  @Field(() => Number)
+  summe: number
+}
+
+@ObjectType()
+export class GdtEntryList {
+  constructor(json: any) {
+    this.state = json.state
+    this.moreEntrysAsShown = json.moreEntrysAsShown
+    this.ownEntries = []
+    if (typeof json.ownEntries !== undefined) {
```

Comment:
Is this really necessary? Why not just:
`this.ownEntries = json.ownEntries ? json.ownEntries : []`

Reply:
When I replace my loop with your code I get this error:
"message": "Expected Iterable, but did not find one for field \"GdtEntryList.ownEntries\".",